genienlp/arguments.py

```diff
@@ -450,17 +457,25 @@ def parse_argv(parser):
     )
 
     parser.add_argument(
-        "--add_types_to_text",
+        "--add_entities_to_text",
         default='no',
         choices=['no', 'insert', 'append'],
-        help='Method for adding types to input text in text-based NER approach',
+        help='Method for adding entities to input text in text-based NER approach',
     )
 
     parser.add_argument(
-        "--add_qids_to_text",
-        default='no',
-        choices=['no', 'insert', 'append'],
-        help='Method for adding qids to input text in text-based NER approach',
+        "--entity_attributes",
+        nargs='+',
+        default=['type_id', 'type_prob', 'qid'],
```

Comment:
What does it mean to set entity_attributes to "type_prob"? Why is this not just "type" and "qid"?

Reply:
'type_prob' and 'type_id' always go together for bootleg cause probability is used for filtering. For embedding-based ned approach `type_prob` is used to weight the type embeddings so it's optional.  
Once we remove the embedding based approach we can remove `type_prob`. For text based approach you can just pass `type_id` or we can set the default here to be `type_id` since we don't use `qid` except in wikidata experiments.

Reply:
Yes the default should be type_id. Do you really need to ablate on type_id without type_prob when you do the embedding experiment?

Reply:
Yes probably